ui/src/components/footer.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2025 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import { useAdminServiceGetApiV1AdminContentManagementSectionsBySectionId } from '@/api/queries';
+import { MarkdownRenderer } from '@/components/markdown-renderer.tsx';
+
+function extractCompanyTag(markdown: string): string | undefined {
+  const match = markdown.match(/:::\s?company\s+(.+?)\s+:::/s);
```

Comment:
I'm a bit late to the party, but why is this *layout* marker named after what it's possible *contents* are? I mean, you could but any information in here, not necessarily related to a company. So why not name it `:::left-align(ed)` or so?

Reply:
Similarly, I would probably rename `:::column` to `:::right-align`. (Or mention "align" first for both -> `:::align-left` and `:::align-right`.)

Reply:
And just like for `:::column` (aka `:::aligh-right`), multiple `:::align-left` (aka `:::company`) should be allowed, if that's not already the case (I did not check the code).